odm2api/ODM2/models.py

```diff
@@ -750,6 +465,12 @@ class Results(Base):
     UnitsObj = relationship(Units)
     VariableObj = relationship(Variables)
 
+    __mapper_args__ = {
+        'polymorphic_on':ResultTypeCV,
+        'polymorphic_identity':'results',
+        'with_polymorphic':'*'
+    }
+
     def __repr__(self):
         return "<Results('%s', '%s', '%s', '%s', '%s')>" % (
             self.ResultID, self.ResultUUID, self.ResultTypeCV, self.ProcessingLevelID, self.ValueCount)
```

Comment:
@emiliom  the mapper_args are just for the polymorphic identity stuff (enables the user to get a TimeSeriesResult back when they query the result table) ResultTypeCV is mentioned because it is the field that determines which type of result is returned. If the  CV is set to 'measurement' then a MeasurementResult type is returned
if you are referring to the __repr__ function, it is just for printing, and i selected a few fields that I thought would help distinguish between different result objects